iModelCore/libsrc/openssl/BeOpenSSL.mke

```diff
@@ -29,9 +29,15 @@ BUILD_NO_STACK_CHECK = 1   # Don't want to modify third party code, but hopefull
 #----------------------------------------------------------
 %include mdl.mki
 
-appName = iTwinOpenSSL
 vendorSrc = $(_MakeFilePath)vendor/
-o = $(OutputRootDir)build/iTwinOpenSSL/
+
+%if defined(iTwinNativeThirdParty)
+  appName = BeOpenSSL

```

Comment:
just want to make sure this is right and not swapped

Reply:
Thanks for checking. The iTwinNativeThirdParty builds the NuGets that are Be*.dll. Imodel-native builds iTwin*.dll. So if iTwinNativeThirdParty is defined, then it builds Be*.dll. Also, the build would fail if the DLL was incorrectly named.

Reply:
the iTwin prefix is not confusing at all haha; thx for explaining